docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -106,17 +111,111 @@ $ pip install -r requirements-dev.txt
 $ pip install setup.py
 ```
 
+After installation, ``xrdsst`` command runs the toolkit, when invoked without any parameters,
+it will give the overview of available options and sub-commands. Sub-commands themselves can
+also have further subcommands, so for example all the supported token operations can be listed
+with ``xrdsst token``:
+
+```
+$ xrdsst token
+usage: xrdsst token [-h] [-v] {init-keys,list,login} ...
+```
+
+
 ## 3 Configuration of X-Road Security Server
 
 ### 3.1 Prerequisites to Configuration
 
-* a central server running the Ubuntu 18.04 LTS or 20.04 LTS operating system, on an x86-64bit platform. 
-* a security server providing management services. 
-* single or multiple security servers(to be configured by the X-Road Security Server Toolkit) running the Ubuntu version 18.04 LTS or 20.04 LTS 
-  or Redhat version 7 or 8 operating system, on an x86-64bit platform or X-Road Security Server Sidecar running in a Docker container
-* configuration file in YAML format for configuring security server
+* X-Road central server. In development, this is best run in the mode where auto-approvals
+are enabled, to be able to register authentication certificates and manage security server
+clients without taking separate actions at central server, see [UG-CS](#Ref_CS-UG) about
+``auto-approve-auth-cert-reg-requests`` and ``auto-approve-client-reg-requests``.
+* Single or multiple security servers to be configured and maintained. Supported and tested
+platforms for the security servers are Ubuntu 18.04/20.04 LTS or Red Hat Enterprise Linux
+(RHEL) 7/8 on an x86-64 platform or
+[X-Road Security Server Sidecar](https://github.com/nordic-institute/X-Road-Security-Server-sidecar)
+running in a Docker container.
+* X-Road security server with subsystem acting as service provider for X-Road management
+services, either in some separate security server, or to be newly added in (one of the)
+security server(s) about to be configured.
```

Comment:
I would go as far as saying that the security server providing management services must be configured and available. Otherwise, we can't register subsystems or auth certificates and many things depend on those.

Reply:
Changed this to:

> * X-Road security server with subsystem acting as service provider for X-Road management
> services, in separate security server.
>